src/handlers/actions/editDomain.ts

```diff
@@ -1,18 +1,17 @@
-import { utils } from 'ethers';
 import { ColonyActionType } from '~graphql';
-import provider from '~provider';
 import { ContractEvent, ContractEventsSignatures } from '~types';
 import {
   toNumber,
   verbose,
   writeActionFromEvent,
   getDomainDatabaseId,
+  transactionHasEvent,
 } from '~utils';
 
 export default async (event: ContractEvent): Promise<void> => {
-  const receipt = await provider.getTransactionReceipt(event.transactionHash);
-  const hasDomainAddedEvent = receipt.logs.some((log) =>
-    log.topics.includes(utils.id(ContractEventsSignatures.DomainAdded)),
+  const hasDomainAddedEvent = await transactionHasEvent(
+    event.transactionHash,
+    ContractEventsSignatures.DomainAdded,
   );
```

Comment:
I know this wasn't introduced here, but I want to mention that I like this pattern/refactor using the `transactionHasEvent` util